ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-isis.yml

```diff
@@ -3,16 +3,23 @@ router_isis:
   instance: EVPN_UNDERLAY
   net: 49.0001.0001.0001.0001.00
   router_id: 192.168.255.3
+  is_type: level-2
+  log_adjacency_changes: true
+  mpls_ldp_sync_default: true
+  timers:
+    local_convergence:
+      protected_prefixes: true
+      delay: 15000
+  advertise:
+    passive_only: true
   no_passive_interfaces:
     - Ethernet1
     - Ethernet2
     - Vlan4093
     - Loopback1
-  is_type: level-2
   address_family: ['ipv4 unicast']
   isis_af_defaults:
     - maximum-paths 2
-  log_adjacency_changes: true
   segment_routing_mpls:
     enabled: true
     router_id: 192.168.255.3
```

Comment:
We need to remove this knob. As we don't support this anymore.

Reply:
Thanks!